napari/components/_tests/test_layer_slicer.py

```diff
@@ -289,3 +292,30 @@ def test_slice_layers_async_with_one_3d_image(layer_slicer):
 
     layer_result = future.result()[layer]
     np.testing.assert_equal(layer_result.data, data[2, :, :])
+
+
+def test_slice_layers_async_with_one_3d_points(layer_slicer):
+    np.random.seed(0)
+    num_points = 100
+    data = np.rint(2.0 * np.random.rand(num_points, 3))
+    layer = Points(data=data)
+    lockable_internal_data = LockableData(layer._data)
+    layer._data = lockable_internal_data
+    dims = Dims(
+        ndim=3,
+        ndisplay=2,
+        range=((0, 3, 1), (0, 3, 1), (0, 3, 1)),
+        current_step=(1, 0, 0),
+    )
+
+    with lockable_internal_data.lock:
+        future = layer_slicer.slice_layers_async(layers=[layer], dims=dims)
+        assert not future.done()
+
+    layer_result = future.result()[layer]
+
+    _indices_view = np.array(layer_result.indices, dtype=int)
+    # get the selected points that are in view
+
+    # Target number of indices manually determined
+    np.testing.assert_equal(len(_indices_view), 48)
```

Comment:
Nit: give the variable a meaningful non-hidden name and you can avoid the explanatory comment.

```suggestion
    points_in_view = np.array(layer_result.indices, dtype=int)

    # Target number of indices manually determined
    np.testing.assert_equal(len(points_in_view), 48)
```